src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer.Common/LanguageServerConstants.cs

```diff
@@ -51,6 +53,15 @@ public static class CodeActions
             public const string CreateComponentFromTag = "CreateComponentFromTag";
 
             public const string AddUsing = "AddUsing";
+
+            public const string Default = "Default";
+        }
+
+        public static class Languages
```

Comment:
Suggest putting this inside the `CodeActions` class. Otherwise it is too generic and could lead to incorrect usage. For example, some places require CSharp language name to be `C#` instead of `CSharp`.